examples/java-matter-controller/java/src/com/matter/controller/commands/common/Command.java

```diff
@@ -190,5 +190,9 @@ public void initArgument(int argIndex, String argValue) {
     mArgs.get(argIndex).setValue(argValue);
   }
 
+  public Object getArgument(int argIndex) {
```

Comment:
We are not supposed to expose the raw object to outside directly, instead , we should use getter helpers to get the filed we want, such as  getArgumentName ...

By the way, I don't see this method get used at all

Reply:
removed, thanks

Reply:
It seems this method is still in the code